dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteRunnable.java

```diff
@@ -784,45 +791,60 @@ private void initTaskQueue() throws StateEventHandleException, CronParseExceptio
         errorTaskMap.clear();
 
         if (!isNewProcessInstance()) {
```

Comment:
It's better to extract a method

Reply:
Yes, but I don't want to do this code refactor in this PR. I will submit another PR to refactor this.